### PR TITLE
Add missing timeseries chart props and functionality

### LINKED ISSUE #189: Missing props in TimeseriesChart
TimeseriesChart needs the following missing props:

- hiddenSeries
- timeseriesColors
- annotations
- ruler
- onMouseMove
- onMouseOut
- onBlur
- series

### PR DESCRIPTION
**Pull request by follan**
I know there is a bit of `any` types here, but they are part of `griff-react`, so it is easy to fix when `griff-react` is typescript ready. I decided that for me to guess the types now will just be unnecessary work.

Fixes #189

### CODE REVIEW

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-192.gearboxjs.preview.cogniteapp.com

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-192.gearboxjs.preview.cogniteapp.com

File: src/components/TimeseriesChart/stories/TimeseriesChart.stories.tsx [hunk 1]
```diff
@@ -222,8 +247,8 @@ storiesOf('TimeseriesChart/Examples', module)
       return (
         <TimeseriesChart
           timeseriesIds={[123]}
-          start={+Date.now() - 30 * 24 * 60 * 60 * 1000}
-          end={Date.now()}
+          startTime={+Date.now() - 30 * 24 * 60 * 60 * 1000}
```
Comment (mrtraser) [inline src/components/TimeseriesChart/stories/TimeseriesChart.stories.tsx]: The same `+` sign here

File: src/components/TimeseriesChart/TimeseriesChart.tsx [hunk 2]
```diff
@@ -99,10 +127,8 @@ export class TimeseriesChart extends React.Component<
               onFetchData={this.onFetchData}
               pointsPerSeries={pointsPerSeries}
               series={griffSeries}
-              timeDomain={[+start, +end]}
-              onFetchDataError={(e: Error) => {
-                throw e;
-              }}
+              timeDomain={[+startTime, +endTime]}
```
[the remark this answers is not in the document]
Comment (follan) [reply src/components/TimeseriesChart/TimeseriesChart.tsx]: Fixed

File: src/components/TimeseriesChart/stories/TimeseriesChart.stories.tsx (hunk 1 above)
Comment (follan) [reply src/components/TimeseriesChart/stories/TimeseriesChart.stories.tsx]: Fixed

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-192.gearboxjs.preview.cogniteapp.com

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-192.gearboxjs.preview.cogniteapp.com

**Pull request merged**